### PR TITLE
po:  Update translate strings of Chinese

### PR DESCRIPTION
**Pull request by Jason23347**
This patch modified the Chinese translations in the po file.

### CODE REVIEW

**Comment by ismaell** [comment]
Great! It would be better to apply this one after the cookies support one.

File: po/zh_CN.po [hunk 1]
```diff
@@ -465,17 +474,17 @@ msgstr "SSL 错误：%s\n"
 #: src/ssl.c:99
 #, c-format
 msgid "SSL error: Certificate error\n"
-msgstr ""
+msgstr "SLL 错误：证书错误\n"
```
Comment (NicodeSS) [inline po/zh_CN.po:477]: 应为
`msgstr "SSL 错误：证书错误\n"`

File: po/zh_CN.po [hunk 2]
```diff
@@ -425,16 +429,21 @@ msgstr ""
...
 "--header=x\t\t-H x\t添加报头字符串\n"
 "--user-agent=x\t\t-U x\t设置用户代理\n"
 "--no-proxy\t\t-N\t不使用任何代理服务器\n"
 "--insecure\t\t-k\t不校验 SSL 证书\n"
+"--no-clobber\t\t-c\t跳过已下载文件\n"
 "--quiet\t\t\t-q\t使用输出简单信息模式\n"
 "--verbose\t\t-v\t更多状态信息\n"
 "--alternate\t\t-a\t另一种进度指示器\n"
 "--help\t\t\t-h\t帮助信息\n"
+"--timeout=x\t\t-T\t设置 I/O 和连接的超时时长\n"
```
Comment (NicodeSS) [inline po/zh_CN.po:446]: 应为
`"--timeout=x\t\t-T x\t设置 I/O 和连接的超时时长\n"`

File: po/zh_CN.po [hunk 3]
```diff
@@ -91,7 +91,7 @@ msgid ""
 "Reactivate connection %d\n"
 msgstr ""
 "\n"
-"重新激活链接:%d \n"
+"重新激活连接:%d \n"
```
Comment (NicodeSS) [inline po/zh_CN.po:94]: 应将冒号换为中文冒号或用空格代替

File: po/zh_CN.po [hunk 4]
```diff
@@ -381,15 +381,19 @@ msgstr ""
...
 "-H x\t添加报头字符串\n"
 "-U x\t设置用户代理\n"
 "-N\t不使用任何代理服务器\n"
 "-k\t不校验 SSL 证书\n"
+"-c\t跳过已下载文件\n"
 "-q\t使用输出简单信息模式\n"
 "-v\t更多状态信息\n"
 "-a\t另一种进度指示器\n"
 "-h\t帮助信息\n"
+"-T\t设置 I/O 和连接的超时时长\n"
```
Comment (NicodeSS) [inline po/zh_CN.po:396]: -T 缺少参数 x

**Comment by Jason23347** [comment]
Thanks for code review of @NicodeSS .

**Comment by ismaell** [comment]
Thanks! Translation needs quite some work in general.

**Pull request closed without merging**